### PR TITLE
removed device integration tutorials

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/device-integration/device-integration-thin-edge.md [hunk 1]
```diff
@@ -7,4 +7,28 @@ section:
...
 
+thin-edge.io is an open-source project to provide a cloud-agnostic edge framework.
+It is much more generic than the device management agent, so it can connect to multiple IoT cloud platforms, and it allows flexible logic being executed on the device.
+It is optimized for a very small footprint and high performance.
+
 To learn more about device integration using thin-edge.io, refer to [thin-edge.io - The open edge framework for lightweight IoT devices](https://thin-edge.io/).
+
+### thin-edge.io on a Raspberry Pi
+
+Follow the tutorial [Getting started with thin-edge.io on a Raspberry Pi](https://thin-edge.github.io/thin-edge.io/start/getting-started/) to get an overview of the installation and configuration of thin-edge.io.
```
Comment (reubenmiller) [inline content/device-integration/device-integration-thin-edge.md:18]: Just a small change as we don't just support Raspberry Pi devices.

```suggestion
Follow the tutorial [Getting started with thin-edge.io](https://thin-edge.github.io/thin-edge.io/start/getting-started/) to get an overview of the installation and configuration of thin-edge.io.
```

File: content/device-integration/device-integration-thin-edge.md [hunk 2]
```diff
@@ -7,4 +7,28 @@ section:
   - device_management
 ---
 
+thin-edge.io is an open-source project to provide a cloud-agnostic edge framework.
+It is much more generic than the device management agent, so it can connect to multiple IoT cloud platforms, and it allows flexible logic being executed on the device.
+It is optimized for a very small footprint and high performance.
+
 To learn more about device integration using thin-edge.io, refer to [thin-edge.io - The open edge framework for lightweight IoT devices](https://thin-edge.io/).
+
+### thin-edge.io on a Raspberry Pi
```
Comment (reubenmiller) [inline content/device-integration/device-integration-thin-edge.md:16]:
```suggestion
### thin-edge.io on a device
```

File: content/device-integration/device-integration-thin-edge.md [hunk 3]
```diff
@@ -7,4 +7,28 @@ section:
...
+
+Follow the tutorial [Getting started with thin-edge.io on a Raspberry Pi](https://thin-edge.github.io/thin-edge.io/start/getting-started/) to get an overview of the installation and configuration of thin-edge.io.
+
+As an example, a Raspberry Pi is used. The Raspberry Pi is a relatively simple and cheap yet powerful device. This makes it ideal for testing and trying out as well as some production use cases.
```
Comment (reubenmiller) [inline content/device-integration/device-integration-thin-edge.md:20]:
```suggestion
As an example, a Raspberry Pi can be used. The Raspberry Pi is a relatively simple and cheap yet powerful device. This makes it ideal for testing and trying out as well as some production use cases.
```

**Review by didier-wenzek — APPROVED** [review]
Approved. Thank you, this will ease maintenance.

**Review by MWindel — APPROVED** [review]

**Pull request merged**